modules/cnf-topology-aware-lifecycle-manager-backup-recovery.adoc

```diff
@@ -0,0 +1,112 @@
+Recovery:: The recovery shuts down containers and uses files from the backup partition to relaunch containers and restore clusters.
+
+.Prerequisites
+
+* Install the {cgu-operator-first}.
+* Provision one or more managed clusters.
+* Install Advanced Cluster Management.
+* Log in as a user with `cluster-admin` privileges.
+* Run an upgrade that is configured for backup.
+
+.Procedure
+
+. Delete the the `ClusterGroupUpgrade` CR "clustergroupupgrades-group-du.yaml".
++
+[source,terminal]
+----
+$ oc delete cgu/du-upgrade-4918 -n ztp-group-du-sno
+----
+
+. Manually log in to the node that you want to recover.
+
+. Check the status of the platform OS deployment by running the following command:
++
+[source,terminal]
+----
+$ oc ostree admin status
+----
+.Example outputs
++
+[source,terminal]
+----
+[root@lab-test-spoke2-node-0 core]# ostree admin status
+* rhcos c038a8f08458bbed83a77ece033ad3c55597e3f64edad66ea12fda18cbdceaf9.0
+    Version: 49.84.202202230006-0
+    Pinned: yes <1>
+    origin refspec: c038a8f08458bbed83a77ece033ad3c55597e3f64edad66ea12fda18cbdceaf9
+----
+<1> The current deployment is pinned. A platform OS deployment rollback is not necessary.
++
+[source,terminal]
+----
+[root@lab-test-spoke2-node-0 core]# ostree admin status
+* rhcos f750ff26f2d5550930ccbe17af61af47daafc8018cd9944f2a3a6269af26b0fa.0
+    Version: 410.84.202204050541-0
+    origin refspec: f750ff26f2d5550930ccbe17af61af47daafc8018cd9944f2a3a6269af26b0fa
+rhcos ad8f159f9dc4ea7e773fd9604c9a16be0fe9b266ae800ac8470f63abc39b52ca.0 (rollback) <1>
+    Version: 410.84.202203290245-0
+    Pinned: yes <2>
+    origin refspec: ad8f159f9dc4ea7e773fd9604c9a16be0fe9b266ae800ac8470f63abc39b52ca
+----
+<1> This platform OS deployment is marked for rollback.
+<2> The previous deployment has been pinned and can be rolled back.
+
+. If you need to trigger a rollback of the platform OS deployment, run the following command:
++
+[source,terminal]
+----
+$ rpm-ostree rollback -r
+----
+
+. The first phase of the recovery shuts down containers, and restores files from the backup partition to the targeted directories. To begin the recovery run the following command:
++
+[source,terminal]
+----
+$ /var/recovery/upgrade-recovery.sh
+----
++
+
+. When prompted, reboot the cluster with the following command:
++
+[source,terminal]
+----
+$ systemctl reboot
+----
+. After the reboot, restart the recovery with the following command:
++
+[source,terminal]
+----
+$ /var/recovery/upgrade-recovery.sh  --resume
+----
+
+[NOTE]
+====
+If the recovery utility fails, you can retry with the `--restart` option:
+[source,terminal]
+----
+$ /var/recovery/upgrade-recovery.sh --restart
+----
+====
+
+.Verification
+To check the status of the recovery run the following command:
+[source,terminal]
+----
+$ oc get clusterversion,co,nodes
+----
+From here you can confirm that:
+
+* The cluster version is available and has the correct version.
+* The cluster operator is availabile.
```

Comment:
Cluster operators will always be available, the field returns true or false. I would say:

`All the cluster operators availability is true`